prow/config.yaml

```diff
@@ -647,23 +647,22 @@ presubmits:
         - "--upload=gs://kubernetes-jenkins/pr-logs"
         env:
         - name: TEST_TMPDIR
-          value: /root/.cache/bazel
+          value: /scratch/.cache/bazel
         # Bazel needs privileged mode in order to sandbox builds.
         securityContext:
           privileged: true
         volumeMounts:
-        - name: cache-ssd
-          mountPath: /root/.cache
+        - name: bazel-scratch
+          mountPath: /scratch/.cache
         ports:
```

Comment:
drop the port and the --git-cache arg now too since we aren't mounting this volume anymore

Reply:
Thanks, fixed!